geles-frontend/types/flower.d.ts

```diff
@@ -26,9 +26,11 @@ interface FlowerInCart {
 }
 
 enum OrderStatus {
-  UNPAID,
-  PAID,
-  DELIVERED
+  UNPAID = "UNPAID",
+  PAID = "PAID",
+  CONFIRMED = "CONFIRMED",
+  DELIVERED = "DELIVERED",
+  CANCELED = "CANCELED"
 }
```

Comment:
Remove enum from here

Reply:
removing it breaks pages where it is used

Reply:
Then reimport the enum from enums.ts file